src/release/breaking-changes/android-java-gradle-migration-guide.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Android Java Gradle migration guide
+description: How to migrate your Android app if you experience a run or build error from Gradle.
+---
+
+## Summary
+
+If you've recently upgraded Android Studio to the Flamingo
+release and have either run or built an existing Android app,
+you might have run into an error similar to the following:
+
+![Error dialog in Android Studio Flamingo]({{site.url}}/assets/images/docs/releaseguide/android-studio-flamingo-error.png){:width="80%"}
+
+The terminal output looks similar to the following:
+
+
+```sh
+FAILURE: Build failed with an exception.
+
+* Where:
+Build file '…/example/android/build.gradle'
+* What went wrong:
+Could not compile build file '…/example/android/build.gradle'.
+> startup failed:
+  General error during conversion: Unsupported class file major version 61
+
+  java.lang.IllegalArgumentException: Unsupported class file major version 61
+  	at groovyjarjarasm.asm.ClassReader.<init>(ClassReader.java:189)
+  	at groovyjarjarasm.asm.ClassReader.<init>(ClassReader.java:170)
+  	[…
+  	 …
+  	 … 209 more lines of Groovy and Gradle stack trace …
+  	 …
+  	 …]
+  	at java.base/java.lang.Thread.run(Thread.java:833)
+```
+
+This error occurs because Android Studio Flamingo
+updates the Java SDK from 11 to 17, which
```

Comment:
updates the _installed_ Java SDK, perhaps?

Reply:
Maybe something along the lines of "Android Studio Flamingo updates its bundled Java SDK version from..."?